utils/types/QuickstartMutationVariable.ts

```diff
@@ -0,0 +1,81 @@
+export interface QuickstartAlertInput {
+  description?: string;
+  displayName: string;
+  rawConfiguration: string;
+  sourceUrl?: string;
+  type: string;
+}
+
+export interface QuickstartDocumentation {
+  displayName: string;
+  url: string;
+  description: string;
+}
+
+export interface DashboardScreenshot {
+  url: string;
+}
+
+export interface QuickstartDashboardInput {
+  description?: string;
+  displayName: string;
+  rawConfiguration: string;
+  sourceUrl?: string;
+  screenshots?: DashboardScreenshot[];
+}
+
+interface QuickstartAuthor {
+  name: string;
+}
+
+export interface QuickstartMetaData {
+  alertConditions?: QuickstartAlertInput[];
+  authors: QuickstartAuthor[];
+  categoryTerms?: string[];
+  description: string;
+  displayName: string;
+  slug?: string;
+  documentation?: QuickstartDocumentation[];
+  icon: string;
+  keywords?: string[];
+  sourceUrl?: string;
+  summary: string;
+  supportLevel?: string;
+  installPlanStepIds?: string[];
+  dashboards?: QuickstartDashboardInput[];
+}
+
+export interface QuickstartMutationVariable {
+  id: string;
+  dryRun: boolean;
+  quickstartMetadata: QuickstartMetaData;
+}
+
+// --------------------- YML / CONFIG Types
```

Comment:
Since the name of this file signifies that this is for _MutationVariables_ we might want to either rename this or move the YAML stuff to it's own file.

Reply:
moved